index.js

```diff
@@ -141,7 +143,9 @@ class SparseIndex extends Index {
     super(byteLength)
     this.chunks = chunks
 
-    binding.quickbit_napi_index_init_sparse(this.handle, this.chunks)
+    binding.quickbit_napi_index_init_sparse(this.handle, this.chunks.map((chunk) => {
+      return { field: asBuffer(chunk.field), offset: chunk.offset }
```

Comment:
can we move this to a hoisted function ? like toBufferChunk, can then be reused above also